README.md

```diff
@@ -348,9 +348,17 @@ You need nodejs 18+
 ### Scripts
```

Comment:
A bit off-topic, probably should be a discussion 

Would you be open to updating the `package.json` script to add the following?
![image](https://user-images.githubusercontent.com/9151470/236571904-608055c9-99c1-4dec-a09d-9cbd4c7424ca.png)
Using `nodemon` and making it easier to override the default `npm start` example that is used

Reply:
@hverlin Sure, i usually use the raw commands, `npx jest` `npx tsc` n' such, so put whatever you like in package.json scripts